backend/src/main/cards/test_cards.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.main.cards.card import Card
```

Comment:
File comment:
This file should instead be placed in `backend/src/test/cards/test_cards.py`